README.md

```diff
@@ -5,44 +5,44 @@ Our open source community's Wiki focuses on answering your questions about learn
 - Curriculum
 - Other Programming Languages
 - Local Campsite Communities
-- Nonprofit Projects  
+- Nonprofit Projects
  ...and much more
```

Comment:
I thing it should be moved to one line without the `...`

Adding a new list element with the `...` wouldn't look good and I don't like using `<br>` on markdown.